vcx/libvcx/src/issuer_credential.rs

```diff
@@ -999,6 +999,17 @@ pub mod tests {
         assert!(!string.is_empty());
     }
 
+    #[test]
+    fn test_empty_val_succeeds() {
```

Comment:
This test actually doesn't check anything.

Reply:
This test checks that empty parameter value does not cause validation failure with empty param